CMakeLists.txt

```diff
@@ -37,3 +49,24 @@ endif()
 
 include(CTest)
 add_test(NAME RunTests COMMAND $<TARGET_FILE:ClaraTests>)
+
+set(target_export_name "${PROJECT_NAME}Targets")
+set(config_install_dir "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+set(version_config "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake")
+
+# install headers
+install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/single_include/"
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+# install project config
+install(TARGETS Clara EXPORT ${target_export_name})
+install(EXPORT ${target_export_name}
+  NAMESPACE Clara::
+  DESTINATION ${config_install_dir}
+  FILE "${PROJECT_NAME}Config.cmake")
+
+# install version config
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(${version_config}
```

Comment:
This needs a workaround as described in onqtam/doctest#225 and catchorg/Catch2#1368.